### ISSUE TITLE
Manganelo no longer working

### ISSUE THREAD

**Issue by donotos** [fmd-project-team/FMD#873]
Hello.

Since 2-3 months https://manganelo.com does not work anymore. it's a clone of mangakalot and i think that it's because they installet an SSL cert. FMD does not seem to accept the redirect to https.

Thanks.

**Comment by Tmp341** [comment]
It seems manganelo is a fork/clone of mangakakalot, as all pictures on it show mangakakalot hosted signs.

**Comment by donotos** [comment]
yeah i think it's a mirror because some of the comics on mangakalot are redirected to manganelo when you read them.

**Comment by dazedcat19** [comment]
some mangakakalot link actually redirected to manganelo. for example this `https://mangakakalot.com/manga/07ghost`
so, I think manganelo was supposed to be the successor of mangakakalot.
